docs/app/pages/components/sections/utilities/color-service/color-service.component.html

```diff
@@ -0,0 +1,229 @@
+                        <div class="color-box grey5-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Grey6</p>
+                        <div class="color-box grey6-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Grey7</p>
+                        <div class="color-box grey7-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Grey8</p>
+                        <div class="color-box grey8-color"></div>
+                    </div>
+                </th>
+            </tr>
+            <tr>
+                <th>
+                    <div class="color-box-container">
+                        <p>Chart1</p>
+                        <div class="color-box chart1-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Chart2</p>
+                        <div class="color-box chart2-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Chart3</p>
+                        <div class="color-box chart3-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Chart4</p>
+                        <div class="color-box chart4-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Chart5</p>
+                        <div class="color-box chart5-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Chart6</p>
+                        <div class="color-box chart6-color"></div>
+                    </div>
+                </th>
+            </tr>
+            <tr>
+                <th>
+                    <div class="color-box-container">
+                        <p>OK</p>
+                        <div class="color-box ok-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Warning</p>
+                        <div class="color-box warning-color"></div>
+                    </div>
+                    <div class="color-box-container">
+                        <p>Critical</p>
+                        <div class="color-box critical-color"></div>
+                    </div>
+                </th>
+            </tr>
+        </tbody>
+    </table>
+</div>
+
+<blockquote>
+    <p><strong>Note</strong>: Refer to the <a routerLink="/css/color-palette">Color Palette</a> page for details about available
+        colors.</p>
+</blockquote>
+
+<p>Below is an example of how the <code>ColorService</code> can be used:</p>
+
+<uxd-snippet language="javascript" [content]="snippets.compiled.appTs"></uxd-snippet>
```

Comment:
Could I suggest a cut-down version of this, for documentation purposes only?

````javascript
import { ColorService, ThemeColor } from 'ux-aspects';
import { Component } from '@angular/core';

@Component({
    selector: 'app',
    templateUrl: './src/app.component.html',
    styleUrls: ['./src/app.component.css']
})
export class AppComponent {

    // ...

    constructor(public colorService: ColorService) {

        this.donutChartColors = [
            {
                backgroundColor: [
                    colorService.getColor('chart1').toRgb(),
                    colorService.getColor('chart2').toRgb(),
                    colorService.getColor('chart3').toRgb(),
                ],
                hoverBackgroundColor: [
                    colorService.getColor('chart1').setAlpha(0.3).toRgba(),
                    colorService.getColor('chart2').setAlpha(0.3).toRgba(),
                    colorService.getColor('chart3').setAlpha(0.3).toRgba(),
                ]
            }
        ];
    }
}
````

I think this would highlight the usage of the color service and the relevant ThemeColor methods.